common/pupil/screening.ts

```diff
@@ -27,3 +28,19 @@ export async function addPupilScreening(pupil: Pupil, screening: PupilScreeningI
 
     logger.info(`Added ${screening.status || 'pending'} screening for pupil ${pupil.id}`, screening);
 }
+
+interface PupilScreeningUpdate {
+    status: PupilScreeningStatus;
+    comment: string;
+}
+
+export async function updatePupilScreening(pupilScreeningId: number, screeningUpdate: PupilScreeningUpdate) {
+    const screening = await prisma.pupil_screening.findFirst({ where: { id: pupilScreeningId } });
+    if (screening === null) {
+        logger.error('cannot find pupil with', { id: pupilScreeningId });
```

Comment:
"Cannot find PupilScreening(${pupilScreeningId}) for update", we generally use "Entity(id)" to make it simpler to find all logs related to one entity.

Reply:
I'll update it 👍 
But we should revisit the setup when refactoring the logging.